tests/phpunit/tests/user.php

```diff
@@ -793,7 +793,6 @@ public function illegal_user_logins() {
 	 */
 	public function test_validate_username_string() {
```

Comment:
Let's add a `@ticket 58311` to the docblock so that folks know this was touched by multiple tickets.